node/flatpak-node-generator.py

```diff
@@ -1405,14 +1385,12 @@ async def generate_package(self, package: Package) -> None:
 
         elif isinstance(source, GitSource):
             repo_name = urllib.parse.urlparse(source.url).path.split('/')[-1]
-            tarball_url = source.tarball_url
-            assert tarball_url is not None
-            metadata = await RemoteUrlMetadata.get(tarball_url, cachable=True)
-            filename = f'{repo_name}-{source.commit}'
+            repo_dir = self.gen.tmp_root / repo_name
```

Comment:
It would be safer to add commit hash to the directory name, since there could be multiple versions of the same module.

Reply:
Adressed.